### DISCUSSION TITLE
home assistant websocket version problem in Node red and HACS intégration impossible with new Home Assistant Green.

### DISCUSSION THREAD

**Discussion by koenroels2** [zachowj/node-red-contrib-home-assistant-websocket#1205] [Beta]
Hi,
When I restart Node-Red: I have this problem and the npm install node-red-contrib-home-assistant-websocket disappears?
I can not install the last version 0.62.1
How resolv this problem. See document join.
thanks
Other problem is if new installation on Home assistant Green. Impossible to install HACS. Consequences: very limited and not possible to integrate Node red compagnon.
[node red home-assistant websocket problem.pdf](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/files/13671519/node.red.home-assistant.websocket.problem.pdf)

best regards
Koen

**Comment by zachowj** [comment]
Version 0.62.1 requires Node-RED 3.1.1+

**Comment by koenroels2** [comment]
Hi,
that's what I expected.
Do you now somebody ho nows how install 3.1.1 on node red in Home Assistant?
thanks

**Reply by zachowj** [reply]
You're going to have to wait until the add-on gets updated.

https://github.com/hassio-addons/addon-node-red

**Comment by koenroels2** [comment]
thanks
